src/lib/Algoan.dto.ts

```diff
@@ -68,3 +96,16 @@ export interface PostBanksUserTransactionDTO {
   simplifiedDescription?: string;
   userDescription?: string;
 }
+
+/**
+ * GET /banks-user/:id/accounts/:account-id/transactions
+ */
+export interface GetBanksUserTransactionDTO extends BanksUserTransaction {
+  organizationId?: string;
+  chatflowId?: string;
+  accountId?: string;
+  inserted?: boolean;
+  validity?: {
+    valid?: boolean;
+  };
+}
```

Comment:
Be careful, this is not part of the DTO.

The DTO file describes a request body/query. You canot query transactions with these parameters

Reply:
I deleted all occurrences of bad DTOs